packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -918,24 +924,12 @@ async function handler(
     })
   );
 
-  // update original rescheduled booking (no seats event)
-  if (!eventType.seatsPerTimeSlot && originalRescheduledBooking?.uid) {
-    await prisma.booking.update({
-      where: {
-        id: originalRescheduledBooking.id,
-      },
-      data: {
-        rescheduled: true,
-        status: BookingStatus.CANCELLED,
-        rescheduledBy: reqBody.rescheduledBy,
-      },
-    });
-  }
```

Comment:
It has been moved to createBooking to make it a part of transaction.

It has happened during testing that new booking creation failed but original booking got cancelled which is inconsistent state.